src/GRA.Web/Views/Home/Dashboard.es.cshtml

```diff
@@ -360,7 +360,23 @@
 
```

Comment:
Badge images on lines 291 & 297 should include the alt-text.

Reply:
Added in commit d9b6231a7e8bcb777845aa3a571fc01ba569d2f8